src/api/lib/Client.ts

```diff
@@ -0,0 +1,138 @@
+import {
+  ClientBuilder,
+  type AuthMiddlewareOptions,
+  type HttpMiddlewareOptions,
+  type AnonymousAuthMiddlewareOptions,
+  type PasswordAuthMiddlewareOptions,
+  type TokenStore,
+  type RefreshAuthMiddlewareOptions,
+} from '@commercetools/sdk-client-v2';
+import { createApiBuilderFromCtpClient } from '@commercetools/platform-sdk';
+import { type clientType } from '../../types/apiClient';
+import { ByProjectKeyRequestBuilder } from '@commercetools/platform-sdk/dist/declarations/src/generated/client/by-project-key-request-builder';
+const PROJECT_KEY = process.env?.REACT_APP_API_PROJECT_KEY as string;
+const projectKey = PROJECT_KEY;
+const HOST = process.env?.REACT_APP_API_HOST_URL as string;
+const AUTH_URL = process.env?.REACT_APP_API_AUTH_URL as string;
+const CLIENT_ID = process.env?.REACT_APP_API_CLIENT_ID as string;
+const CLIENT_SECRET = process.env?.REACT_APP_API_CLIENT_SECRET as string;
+const scopes = [process.env?.REACT_APP_API_SCOPES as string];
+
+const authMiddlewareOptions: AuthMiddlewareOptions = {
```

Comment:
I thought we will use RTK query. But it's ok. I will have opportunity to work with simple redux :)